tests/backup.test.js

```diff
@@ -25,45 +25,22 @@ describe("BackupRecoveryKey", () => {
     test("create from base64 string", () => {
         const backupkey = BackupDecryptionKey.fromBase64("Ha9cklU/9NqFo9WKdVfGzmqUL/9wlkdxfEitbSIPVXw");
 
-        const decypted = backupkey.decryptV1(
-            encryptedMegolm.session_data.ephemeral,
-            encryptedMegolm.session_data.mac,
-            encryptedMegolm.session_data.ciphertext,
+        const decrypted = JSON.parse(
+            backupkey.decryptV1(
+                encryptedMegolm.session_data.ephemeral,
+                encryptedMegolm.session_data.mac,
+                encryptedMegolm.session_data.ciphertext,
+            ),
         );
 
-        expect(decypted.algorithm).toStrictEqual(aMegolmKey.algorithm);
-        expect(decypted.sender_key).toStrictEqual(aMegolmKey.sender_key);
-        expect(decypted.session_key).toStrictEqual(aMegolmKey.session_key);
-    });
-
-    test("create export and import base58", () => {
```

Comment:
It would be nice to keep support for base58 since some clients (namely Element Web) ask for the base58 format of the decryption key when restoring a backup.

Reply:
> some clients (namely Element Web) ask for the base58 format of the decryption key when restoring a backup.

I don't *think* they do, but I could well be missing something. Can you give me some more clues as to how to find this flow in the app?

Reply:
The way to get this flow in Element Web is to use an account that skipped all identity/crypto/cross-signing setup (a brand new account is the easiest way to test that), then visit use Settings > Security & Privacy > Connect to Key Backup, which will have it ask for the base58 room backup decryption key instead of the 4S passphrase/key.

However, it's very easy to lose this flow, because once you've set up cross-signing / 4S, that button asks you for the 4S passphrase/key and there's seemingly no way to make it go back to asking for the backup decryption key.